database.sql

```diff
@@ -579,6 +579,27 @@ CREATE TABLE IF NOT EXISTS `delayed-post` (
 	FOREIGN KEY (`wid`) REFERENCES `workerqueue` (`id`) ON UPDATE RESTRICT ON DELETE CASCADE
 ) DEFAULT COLLATE utf8mb4_general_ci COMMENT='Posts that are about to be distributed at a later time';
 
+--
+-- TABLE delivery-queue
+--
+CREATE TABLE IF NOT EXISTS `delivery-queue` (
+	`gsid` int unsigned NOT NULL COMMENT 'Global Server ID',
+	`uri-id` int unsigned NOT NULL COMMENT 'Id of the item-uri table entry that contains the item uri',
+	`created` datetime COMMENT '',
+	`command` varbinary(32) COMMENT '',
+	`cid` int unsigned COMMENT 'contact_id (ID of the contact in contact table)',
```

Comment:
This entire comment can be inferred just from the foreign key statement below, instead please explain what this contact is about. Is it the delivery target contact?
```suggestion
	`cid` int unsigned COMMENT 'Delivery target contact ID',
```

Reply:
Done